helix-core/src/main/java/org/apache/helix/util/InstanceValidationUtil.java

```diff
@@ -55,6 +55,7 @@
       ImmutableSet.of(HelixDefinedState.DROPPED.name(), HelixDefinedState.ERROR.name());
 
   static final String UNHEALTHY_PARTITION = "UNHEALTHY_PARTITION";
+  static final String HOST_NO_STATE_ERROR = "HOST_NO_STATE_ERROR:";
```

Comment:
Is this a prefix thing? For human readable, please have an empty space after ":".

Reply:
Thanks for the suggestion. All the other error messages are the same pattern with ":". I would prefer to keep the current pattern for consistency.